docs/adr/adr-010-incentivized-testnet-monitoring.md

```diff
@@ -167,14 +168,44 @@ The Prometheus docs state the following with regard to [Denial of Service](https
 
 So if we are concerned about the public leaderboard crashing the Prometheus instance that we use for internal dashboards, we may want to host two separate instances. This seems feasible by configuring OTEL Collector to export to two different Prometheus instances. This is a one way door, I suggest sticking with one instance because Grafana Cloud guarantees 99.5% uptime.
 
-### Why doesn't the Celestia team host OTEL Collectors for node operators?
+### Which actor should run OTEL Collector(s) during the incentivized testnet?
 
-1. Node operators will lose the ability to monitor their own celestia-node. Since opentelemetry-go only supports configuring one exporter ( [open-telemetry/opentelemetry-go#3055](https://github.com/open-telemetry/opentelemetry-go/issues/3055)), if node operators were obligated to export metrics to a Celestia team managed OTEL Collector endpoint, they wouldn't be able to export to their own OTEL Collector (and by proxy any telemetry platforms they wish to use). This violates the desideratum:
+#### Scenario A: Node operators
+
+Pros
+
+- This deployment architecture is more representative of mainnet where node operators will run their own telemetry stack to monitor their node. Exposing node operators to OTEL Collector during incentivized testnet allows them to practice this deployment architecture prior to mainnet.
```

Comment:
The header specifies the incentivized testnet while the pros are related to mainnet. A bit incoherent

Reply:
I think that incentivized testnet is related to mainnet because it allows node operators to practice their mainnet setup during incentivized testnet.

Do you have suggestions to make this more coherent?

Reply:
I meant the mainnet and incentivized testnet are different scenarios that have different pros and cons. They have common things and have differences like incentivization which affects the list. Nevertheless, I think it is fine to keep things as they are, as we decided. Can be resolved